src/tiledb/cloud/soma/ingest.py

```diff
@@ -103,44 +103,13 @@ def run_ingest_workflow_udf(
     # https://github.com/TileDB-Inc/TileDB-Cloud-Py/pull/512
 
     logger = get_logger_wrapper(level=logging_level)
-    logger.debug("ENUMERATOR ENTER")
-    logger.debug("ENUMERATOR INPUT_URI  %s", input_uri)
-    logger.debug("ENUMERATOR OUTPUT_URI %s", output_uri)
-    logger.debug("ENUMERATOR DRY_RUN    %s", str(dry_run))
 
     h5ad_ingest = None
     collector = None
 
     vfs = tiledb.VFS(config=extra_tiledb_config)
 
-    if vfs.is_file(input_uri):
-        logger.debug("ENUMERATOR VFS.IS_FILE")
-
-        name = ("dry-run" if dry_run else "ingest") + "-h5ad-file"
-
-        grf = dag.DAG(
-            name=name,
-            mode=dag.Mode.BATCH,
-            namespace=carry_along.get("namespace", namespace),
-        )
-
-        h5ad_ingest = grf.submit(
-            _ingest_h5ad_byval,
-            output_uri=output_uri,
-            input_uri=input_uri,
-            measurement_name=measurement_name,
-            extra_tiledb_config=extra_tiledb_config,
-            ingest_mode=ingest_mode,
-            platform_config=platform_config,
-            resources=carry_along.get("resources", resources),
-            access_credentials_name=carry_along.get("access_credentials_name", acn),
-            logging_level=logging_level,
-            dry_run=dry_run,
-        )
-
-    elif vfs.is_dir(input_uri):
-        logger.debug("ENUMERATOR VFS.IS_DIR")
-
+    if vfs.is_dir(input_uri):
```

Comment:
Here's the fix! `vfs.is_file(input_uri)` returns `True`, erroneously. We need to always test with `is_file()` first.